.env.dist

```diff
@@ -98,7 +98,8 @@ DTT_BASE_URL=http://web:8080
 DTT_API_URL=http://selenium:4444/wd/hub
 DTT_MINK_DRIVER_ARGS="['chrome', null, 'http://selenium:4444/wd/hub']"
 
-# Wether to restore the production databases when Docker containes are up. Copy
-# this line in .env file and set 'yes' as value if you want to restore the
+# This environment variable is used as a switch to wake up the containers with
+# empty databases (value 'no') or with databases importing data from production.
+# Copy this line in .env file and set 'yes' as value if you want to restore the
```

Comment:
No. It's `.env` because that provides vars to host machine shell. Then we list the var in each service w/o value === _take the value from host machine_